<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeView</class>
 <widget class="QFrame" name="NodeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-1</y>
     <width>265</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>Node</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="numberText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>45</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identifierText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifier</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections>
  <connection>
   <sender>descriptionText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NodeView</receiver>
   <slot>updateEntityDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>updateEntityDescription()</slot>
 </slots>
</ui>
